emotionalTesting/FromBackgroundView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  FromBackgroundView.swift
+//  emotionalTesting
+//
+//  Created by Juan Harrington on 6/08/22.
+//
+import SwiftUI
+struct ReusableView: View {
+    
+    var backgroundColor: [Double] = [0, 0, 0]
```

Comment:
```suggestion
    var backgroundColor: Color
```

Reply:
Thank you for feedback, it is actually done you can check it on the new PR